src/pages/User/validations/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as Yup from 'yup';
+
+export const signUpSchema = Yup.object().shape({
+  firstName: Yup.string().required(),
```

Comment:
LIke you can declare error msg here like below
firstName: Yup.string().required('First Name is Required'),